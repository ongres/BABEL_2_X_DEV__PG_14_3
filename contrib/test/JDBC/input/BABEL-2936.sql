CREATE TABLE dbo.t2936 (c XML NULL);
go

exec sp_describe_undeclared_parameters N'INSERT INTO [dbo].[t2936]([c]) values (@P1)'
go

DROP TABLE dbo.t2936;
go
