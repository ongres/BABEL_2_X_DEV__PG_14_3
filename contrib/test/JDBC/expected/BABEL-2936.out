CREATE TABLE dbo.t2936 (c XML NULL);
go

exec sp_describe_undeclared_parameters N'INSERT INTO [dbo].[t2936]([c]) values (@P1)'
go
~~START~~
int#!#varchar#!#int#!#nvarchar#!#smallint#!#tinyint#!#tinyint#!#int#!#varchar#!#varchar#!#varchar#!#nvarchar#!#int#!#varchar#!#varchar#!#varchar#!#bit#!#bit#!#bit#!#bit#!#bit#!#varchar#!#int#!#int
1#!#@p1#!#241#!#xml#!#-1#!#0#!#0#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#0#!#0#!#0#!#1#!#0#!#<NULL>#!#241#!#8100
~~END~~


DROP TABLE dbo.t2936;
go
